source/shared/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -248,7 +254,7 @@ For the app to work properly with the current version of the Desktop Wallet, mak
 
 #. Navigate to where you have downloaded the Ledger install package. For example, this might be ``~/Downloads/concordium-ledger-app-2.0.1-target-2.0.0``. To navigate to this directory in a Terminal, enter ``cd ~/Downloads/concordium-ledger-app-2.0.1-target-2.0.0``.
 
-#. Run the ``install.sh`` file from the folder that you extracted the files to.
+#. Run the ``./install.sh`` file from the folder that you extracted the files to.
```

Comment:
```suggestion
#. Install the Concordium application on the Ledger by running the following script from the folder you extracted the files to:

   .. code-block:: console

      ./install.sh

```

Reply:
I think this is easier for people to understand (I just copied the explanation from the install guide)